stl/inc/format

```diff
@@ -3606,9 +3606,9 @@ struct _Format_checker {
         : _Parse_context(_Fmt, _Num_args, _Arg_type),
           _Parse_funcs{&_Compile_time_parse_format_specs<_Args, _ParseContext>...} {}
     constexpr void _On_text(const _CharT*, const _CharT*) const noexcept {}
-    constexpr void _On_replacement_field(const size_t _Id, const _CharT*) const {
-        _ParseContext _Parse_ctx({});
-        (void) _Parse_funcs[_Id](_Parse_ctx);
+    constexpr void _On_replacement_field(const size_t _Id, const _CharT* _Last) {
+        _Parse_context.advance_to(_Parse_context.begin() + (_Last - &*_Parse_context.begin()));

```

Comment:
Instead of `&*_Parse_context.begin()`, right line 3614 below says `_Parse_context.begin()._Unwrapped()`. We should follow that precedent. Occurs again on 3647. I see that this was likely copied from pre-existing inconsistency on 3655, let's change that one too.